roles/comanage-registry/defaults/main.yml

```diff
@@ -149,4 +149,7 @@ comanage_ldap_provisioner_entitlements:
 #    filename: "registry"
 #    user: root
 #    job: |
-#      cd {{ path_to_registry }}/app && Console/cake action
\ No newline at end of file
+#      cd {{ path_to_registry }}/app && Console/cake action
+
+# Whether to install RCIAM util cron jobs; Enabled by default
```

Comment:
Fix name in comment:
`# Whether to install COmanage Registry cron jobs; Enabled by default`

Reply:
Done